### PR TITLE
Modifications to run tests locally

### PR DESCRIPTION
**Pull request by sara-02**
Add steps to run test locally.

### CODE REVIEW

**Comment by pkajaba** [comment]
[test]

File: README.md [hunk 1]
```diff
@@ -4,4 +4,33 @@ License Analysis Service analyzes the given stack and returns the following:
...
+* set export PYTHONPATH=`pwd`/src
+
+* gunicorn -b 0.0.0.0:$SERVICE_PORT rest_api:app
+
+* curl localhost:<SERVICE_PORT> returns `{status: ok}`
+
+
+## How to test locally:
+
+* Set the value of `MAJORITY_THRESHOLD` in the env.
```
Comment (pkajaba) [inline README.md]: Can't you create script `runtest.sh` and do these operations (including setting `DATA_DIR`)?

Comment (sara-02) [reply README.md]: @pkajaba Yes, we can create a script. However the current set of instructions worked for me, all tests are passing.

Comment (pkajaba) [reply README.md]: I did not set `MAJORITY_THRESHOLD`. However, you are writing here: https://github.com/fabric8-analytics/fabric8-analytics-license-analysis/pull/21/files#diff-04c6e90faac2675aa89e2176d2eec7d8R32
> By default the value of `MAJORITY_THRESHOLD` used is 0.6.

**Comment by sara-02** [comment]
@pkajaba I have added the scripts, PTAL.

**Pull request merged**